src/entities/user.ts

```diff
@@ -56,11 +64,9 @@ export class User extends BaseEntity {
   @Column({ unique: true })
   walletAddress?: string;
 
-  @Field({ nullable: true })
```

Comment:
@mohammadranjbarz @CarlosQ96 Why we don't remove `password` and `encryptedPassword` field completely? They remained from impact-graph legacy project and don't have any meaning in Giveth.

Reply:
We use `encryptedPassword` for admin users, and about password I thin if delete it now we should change admin bro create admin page to replace password field with another field